item.rb

```diff
@@ -0,0 +1,35 @@
+require 'date'
+
+# Item class
+class Item
```

Comment:
- [ ] Item class should have have one Genre, Author, Label, 
```ruby
    attr_reader :genre
    attr_reader :author
    attr_reader :label
```

- [ ] Each class from the [UML class diagram](https://github.com/microverseinc/curriculum-ruby/blob/main/groupl-capstone/images/catalog_of_my_things.png) should be created in a separate .rb file: Actually you are missing the `author `class.

Reply:
Hi, as specified in the requirements of the project (divided), the author class is part of the requirements for team member #3 and we are only 2, that's why we didn't do it, can you please take into consideration this? Thank you so much